src/libraries/System.Net.WebSockets/src/System/Net/WebSockets/ManagedWebSocket.cs

```diff
@@ -1495,9 +1490,7 @@ private static unsafe int ApplyMask(Span<byte> toMask, int mask, int maskIndex)
 
                 if (toMaskEnd - toMaskPtr >= sizeof(int))
                 {
-                    int rolledMask = BitConverter.IsLittleEndian ?
-                        (int)BitOperations.RotateRight((uint)mask, maskIndex * 8) :
-                        (int)BitOperations.RotateLeft((uint)mask, maskIndex * 8);
+                    int rolledMask = (int)BitOperations.RotateRight((uint)mask, maskIndex * 8);
```

Comment:
I think we should keep the old implementation here, see my comment about the mask packing.

Reply:
Sure, I was on the fence about this one.